### PR TITLE
Add experiments create endpoint

### PR DESCRIPTION
**Pull request by smacker**
Implements endpoint needed for https://github.com/src-d/code-annotation/issues/160

### CODE REVIEW

File: server/repository/experiments.go [hunk 1]
```diff
@@ -69,3 +70,19 @@ func (repo *Experiments) GetAll() ([]*model.Experiment, error) {
 
 	return results, nil
 }
+
+// Create experiment model in database
```
Comment (carlosms) [inline server/repository/experiments.go]: I'd mention that the new ID is also set.

`// Create stores an Experiment in database. On success the assigned ID is set`

**Review by carlosms — APPROVED** [review]
LGTM

**Review by bzz — APPROVED** [review]
lgtm

👍 for having tests

Probably a nit-pick, but having changes with refactoring in a separate commit from the changes that adds new functionality would make this PR even better.

**Comment by smacker** [comment]
rebase, squshed, merging after CI is green

**Pull request merged**